CHANGELOG.md

```
@@ -1,3 +1,8 @@
+## [v1.8.1](https://github.com/sQVe/grove/releases/tag/v1.8.1) - 2026-03-16
+
+### Fixed
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)